docs/src/running-tests-java.md

```diff
@@ -0,0 +1,16 @@
+---
+id: running-tests
+title: "Running Tests"
+---
+
+Playwright tests can be run in a variety of ways. We recommend hooking it up to your favorite test runner, e.g. [JUnit](./test-runners.md) since it gives you the ability to run tests in parallel, run single test, etc.
```

Comment:
We need a JUnit example here or just drop this page altogether as it doesn't add any value and doesn't actually help me integrate pw with my test runner.